allsky/README.md

```diff
@@ -2,3 +2,10 @@ All-sky
 =======
 
 * ``allsky_rosat.fits``: [ROSAT](http://heasarc.gsfc.nasa.gov/docs/rosat/rass.html) Soft X-ray Diffuse Background in the 3/4 keV band, in an Aitoff projection
+
+* ``ligo_simulated.fits.gz``:
+  [All-sky posterior map](http://www.ligo.org/scientists/first2years/) in
+  HEALPIX format for a simulated event observed by the Advanced Laser
+  Interferometer Gravitational-wave Observatory (aLIGO). The event ID is
+  890025 and the map was produced by Bayestar. The maps are described in more
+  detail in [Berry et al. (2014)](http://arxiv.org/abs/1411.6934).
```

Comment:
This particular data file is from `[Singer et al. (2014)](http://arxiv.org/abs/1404.5623)`.